### PR TITLE
fix: typo "hierarchy", "should"

### PR DESCRIPTION
**Pull request by PabloLION**
This only changes the typo of `Hiearchy` to `Hierarchy` and `hiearchy` to `hierarchy`
More typos: "shoud" -> "should"

# Community PR Review Checklist

## First Time Specifics

- [x] if its your first pull request to Dendron, watch out for the [CLA](https://en.wikipedia.org/wiki/Contributor_License_Agreement) bot that will ask you to agree to Dendron's CLA
- [x] if its your first pull request and you're on our Discord, make sure that Kevin gives you the [horticulturalist](https://wiki.dendron.so/notes/7c00d606-7b75-4d28-b563-d75f33f8e0d7.html#horticulturalist) role by adding your discord as part of the description  👨‍🌾👩‍🌾

## Commit

- [x] make sure your branch names adhere to our [branch style](https://docs.dendron.so/notes/adc39825-77a6-46cf-9c49-2642fcb4248e.html#branch-style)

## Code

- [x] code should follow [Code Conventions](https://docs.dendron.so/notes/adc39825-77a6-46cf-9c49-2642fcb4248e.html#branch-style)
- [x] circular dependency check: make sure your code is not introducing new circular dependencies in plugin-core.  See [Avoiding Circular Dependencies](https://docs.dendron.so/notes/adc39825-77a6-46cf-9c49-2642fcb4248e.html#commit-style).
- [x] sticking to existing conventions instead of creating new ones
  - eg: [if configuration for utilities are already in one module or package, add future utilities there as well](https://github.com/dendronhq/dendron/pull/1960#discussion_r786228021)

## Tests

- [x] [Confirm existing tests pass](https://docs.dendron.so/notes/99q7A73uGmCwu2KvSHZro.html#executing-tests)
- [x] [Confirm manual testing](https://docs.dendron.so/notes/99q7A73uGmCwu2KvSHZro.html#manual-testing)

### CODE REVIEW

**Comment by PabloLION** [comment]
It seems that my "sort import" changed import orders.. should I change it back?

File: packages/common-all/src/types/workspace.ts [hunk 1]
```diff
@@ -276,7 +276,7 @@ export type DendronConfig = {
   useKatex?: boolean;
 
   /**
-   * Shoud show hiearchy
```
Comment (SeriousBug) [inline packages/common-all/src/types/workspace.ts:279]: Could you also fix `Should` here?

File: packages/engine-test-utils/CHANGELOG.md [hunk 2]
```diff
@@ -12243,7 +12243,7 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 ### Bug Fixes
 
-- handle single hiearchy properly ([3e822a6](https://github.com/dendronhq/dendron/commit/3e822a6d659c5f4da6bd3ddad9bf1d93a5c353e3))
+- handle single hierarchy properly ([3e822a6](https://github.com/dendronhq/dendron/commit/3e822a6d659c5f4da6bd3ddad9bf1d93a5c353e3))
```
Comment (SeriousBug) [inline packages/engine-test-utils/CHANGELOG.md:12246]: I think the changelog should stay as-is, it reflects what the release notes were at the time.

File: packages/plugin-core/CHANGELOG.md [hunk 3]
```diff
@@ -18133,7 +18133,7 @@ The changelog below lists changes that follow the [conventional commits](https:/
 
 ### Bug Fixes
 
-- handle single hiearchy properly ([3e822a6](https://github.com/dendronhq/dendron/commit/3e822a6d659c5f4da6bd3ddad9bf1d93a5c353e3))
+- handle single hierarchy properly ([3e822a6](https://github.com/dendronhq/dendron/commit/3e822a6d659c5f4da6bd3ddad9bf1d93a5c353e3))
```
Comment (SeriousBug) [inline packages/plugin-core/CHANGELOG.md:18136]: Same with this one, let's leave the changelogs untouched

**Review by SeriousBug — COMMENTED** [review]
Thanks for the correction! I left a messages.

**Comment by jonathanyeung** [comment]
lgtm, thanks!

File: packages/common-all/src/types/workspace.ts (hunk 1 above)
Comment (PabloLION) [reply packages/common-all/src/types/workspace.ts:279]: sure

File: packages/engine-test-utils/CHANGELOG.md (hunk 2 above)
Comment (PabloLION) [reply packages/engine-test-utils/CHANGELOG.md:12246]: I think the purpose of "change log" is not only to show the dev progress (when we have it in git commit message) but also to search? And when we search, ideally we won't write a wrong spelling?
Maybe you want to keep it for the git blame or some other reason. I don't know which is better, could you reconsider about this? And we do whatever you decided.

Comment (SeriousBug) [reply packages/engine-test-utils/CHANGELOG.md:12246]: Makes sense, we can keep it in.

**Pull request merged**